docs/docs/developers/reference/environment_reference/cli_wallet_reference.md

```diff
@@ -208,7 +208,7 @@ Generates a secret key and deploys an account contract. Uses a Schnorr single-ke
 - `--skip-initialization`: Skip initializing the account contract. Useful for publicly deploying an existing account.
 - `--public-deploy`: Publishes the account contract instance (and the class, if needed). Needed if the contract contains public functions.
 - `-p, --public-key <string>`: Public key that identifies a private signing key stored outside of the wallet. Used for ECDSA SSH accounts over the secp256r1 curve.
-- `-n, --node-url <string>`: URL of the PXE (default: "http://host.docker.internal:8080")
+- `-n, --node-url <string>`: URL of Aztec Node (default: "http://host.docker.internal:8080")
```

Comment:
Forgot to update in the last PR. These references seems like a pretty bad thing to have given that `aztec --help` is available and there it will not become stale.